src/3rd_party-static/jsoncpp/include/json/reader.h

```diff
@@ -135,9 +135,9 @@ namespace Json {
       bool readCppStyleComment();
       bool readString();
       void readNumber();
-      bool readValue();
-      bool readObject( Token &token );
-      bool readArray( Token &token );
+      bool readValue( int *const depth_level );
```

Comment:
@ZhdanovP What the reason use "int *const", why you don't use "int &"?
It's much easier for understanding, isn't?